test/function_call_test.js

```diff
@@ -82,19 +82,32 @@ describe('function calls', () => {
     `);
   });
 
-  it.skip('places parentheses in calls with multi-line function arguments after the closing brace', () => {
+  it('places parentheses in calls with multi-line function arguments after the closing brace', () => {
     check(`
       promise.then ->
+        a()
         b # c
       d
     `, `
       promise.then(function() {
+        a();
         return b; // c
       });
       d;
     `);
   });
 
+  it('places parentheses in calls with fat arrow function arguments after the closing brace', () => {
```

Comment:
This is due to esnext identifying it as a candidate for transforming into an arrow function, since it is just a function with a single return statement. I can merge and clean up after.